sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -187,12 +192,14 @@ def send(self, events, **kwargs):
         :keyword str content_type: The type of content to be used to send the events.
          Has default value "application/json; charset=utf-8" for EventGridEvents,
          with "cloudevents-batch+json" for CloudEvents
+        :keyword str channel_name: Optional. Required only when publishing to partner namespaces with partner topic.
```

Comment:
I would rephrase to avoid using the word "required" in the "Required only" disclaimer. Maybe something like "Used to specify the name of event channel when publishing to partner namespaces with (a?) partner topic"